### PR TITLE
Fix cross origin requests

### PR DESCRIPTION
**Pull request by cavis**
Adding `headers: :any` to allow non-whitelisted domains to do the preflight `Access-Control-Request-Method: GET`.

Also whitelisted the `prx.tech` domain.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -39,13 +39,13 @@ class Application < Rails::Application
...
       allow do
-        origins /.*\.prx\.(?:org|dev)$/
+        origins /.*\.prx\.(?:org|dev|tech)$/
         resource '/api/*', methods: [:get, :put, :post, :options], headers: :any
       end
 
       allow do
         origins '*'
-        resource '/api/*', methods: [:get]
+        resource '/api/*', methods: [:get], headers: :any
```
Comment (chrisrhoden) [inline config/application.rb]: I can't merge this – we actually don't know what the implications of allowing anyone to make authenticated requests against the API are. We really just want to let them make unauthenticated public requests.

Comment (cavis) [inline config/application.rb]: So except for unauthorized GET requests, we're keeping the CMS api internal-only for the time being?  That's fine - just want to be clear.

Comment (chrisrhoden) [inline config/application.rb]: For now, yeah. I mean, there's not really a way for anyone else to get tokens at this point anyway, so it's sort of moot. Just don't want to open this up until it's necessary.

**Comment by cavis** [comment]
There we go!  Lines changed: 1.

**Pull request merged**